### PR TITLE
TernaryOperatorSpacesFixer - fix for alternative control structures

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5224

Pinging @Riimu (as bug reporter) for review.

### CODE REVIEW

**Review by GrahamCampbell — APPROVED** [review]

**Comment by Riimu** [comment]
I believe the list is missing `T_ELSE` and `T_ELSEIF`. I.e. to cover cases like:

```php
<?php

if (true ? true : false):
  echo "Hello World\n";
elseif (true):
  echo "Second Hello\n";
else:
  echo "Last Hello\n";
endif;
```

**Comment by kubawerlos** [comment]
@Riimu definitely, fixed, thanks for spotting this.

**Comment by GrahamCampbell** [comment]
This PR doesn't seem to merge up to master cleanly. I think a separate additional PR needs to be prepared.

File: src/Fixer/Operator/TernaryOperatorSpacesFixer.php [hunk 1]
```diff
@@ -118,6 +122,32 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         }
     }
 
+    /**
+     * @param int $index
+     *
+     * @return bool
+     */
+    private function belongsToAlternativeSyntax(Tokens $tokens, $index)
```
Comment (GrahamCampbell) [inline src/Fixer/Operator/TernaryOperatorSpacesFixer.php:130]: static?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Fixer/Operator/TernaryOperatorSpacesFixer.php (hunk 1 above)
Comment (kubawerlos) [reply src/Fixer/Operator/TernaryOperatorSpacesFixer.php:130]: nah, `belongsToGoToLabel` is not static either

**Comment by Riimu** [comment]
Tried this against the test case and the company project where I first noticed the issue. No more false positives! Thanks for taking time to fix this.

So, LGTM ✅

**Review by Riimu — APPROVED** [review]

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**